variables.sh

```diff
@@ -0,0 +1,38 @@
+# This should be included in both scripts, not invoked directly
```

Comment:
Unrelated; revert file.

Reply:
Yeah, I took the opportunity to remote build `local_test_release` and check if per-build target was indeed working. And it actually didn't, and I'm still trying to understand why:

https://app.staging.yourbase.io/yourbase/yb/builds/64653c0d-58e8-4b8d-ac41-8ce7856c1c54